airflow/models.py

```diff
@@ -3057,8 +3190,8 @@ def delete(cls, xcoms, session=None):
             xcoms = [xcoms]
         for xcom in xcoms:
             if not isinstance(xcom, XCom):
-                raise TypeError(
-                    'Expected XCom; received {}'.format(type(xcom)))
+                raise TypeError('Expected XCom; received {}'.format(
+                    xcom.__class__.__name__))
```

Comment:
Nit: PEP8 enforces alignment like this, I think:

+                raise TypeError('Expected XCom; received {}'.format(
+                                          xcom.__class__.__name__))